### PR TITLE
Update minikube to 1.14 and also update container images to latest

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@Madhu-1 PTAL .

**Comment by humblec** [comment]
@Madhu-1 ptal .. thanks !

File: scripts/minikube.sh [hunk 1]
```diff
@@ -53,7 +53,7 @@ function install_kubectl() {
 
 # configure minikube
 MINIKUBE_VERSION=${MINIKUBE_VERSION:-"latest"}
-KUBE_VERSION=${KUBE_VERSION:-"v1.13.0"}
+KUBE_VERSION=${KUBE_VERSION:-"v1.14.1"}
```
Comment (Madhu-1) [inline scripts/minikube.sh]: @humblec can you update this one also to `1.14.2`?

**Comment by humblec** [comment]
@Madhu-1 updated to `1.14.2` version. PTAL.

**Comment by humblec** [comment]
@Madhu-1 Thanks , merging this.

**Pull request merged**